xml/System/GCMemoryInfo.xml

```diff
@@ -279,9 +279,9 @@ The memory after `OBJ_D` is not considered part of the `FragmentedBytes` but is
         <ReturnType>System.Int64</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the memory load when the last garbage collection occurred.</summary>
-        <value>The memory load when the last garbage collection occurred.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the physical memory load when the last garbage collection occurred.</summary>
+        <value>The physical memory load, in bytes, when the last garbage collection occurred.</value>
+        <remarks>If you don't touch the pages of an object, the memory load doesn't change significantly.</remarks>
```

Comment:
I would probably say this -

"Data is only brought into physical memory on first touch so if you allocated a big object but doesn't actually use it, most of its memory will not be in physical memory so you could see the memory load change much."